sema/ro/creator/api.py

```
@@ -0,0 +1,89 @@
+    @property
+    def name(self) -> str:
+        """The name of this strategy. Used as key in the registry."""
+        return self._name
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc):
+        pass
+
+    def __call__(self, uses: dict[str, any]) -> RocStrategy:
```

Ruff (0.8.2)

Unused method argument: `uses`

(ARG002)